aws/resource_aws_appsync_graphql_api.go

```diff
@@ -159,6 +165,11 @@ func resourceAwsAppsyncGraphqlApiCreate(d *schema.ResourceData, meta interface{}
 		return err
 	}
 
+	err = resourceAwsAppsyncSchemaPut(*resp.GraphqlApi.ApiId, d, meta)
```

Comment:
This handling should be removed below the `d.SetId()` call so Terraform does not lose track of the AppSync GraphQL API resource if this errors.